rebound/__init__.py

```diff
@@ -15,9 +15,11 @@
 
 from .librebound import ReboundModule
 from .particle import Particle
+from .particle import Orbit
 from .interruptible_pool import InterruptiblePool
 import sys
 module = ReboundModule(__name__)
 module.Particle = Particle
+module.Orbit = Orbit
```

Comment:
BTW, I thought about this, but couldn't come up with a scenario where one would need it. But it's ok to leave it in.